targets/TARGET_ARM_SSG/TARGET_CM3DS_MPS2/device/drivers/TZ-TRNG/.mbedignore

```diff
@@ -0,0 +1,10 @@
+host/src/tests/tztrng_test/tztrng_test.c
```

Comment:
these files were added because of easier integration? Not running any script that would take the release of trng driver, and would take only files that are relevant (similar is for makefiles and other files that are "unused" here .

Reply:
That file was added to prevent the compilation of "unused" code in mbed-os but still keep all the file from the TZ-TRNG open-source driver for easy maintenance.
I did not understand your comment: do you suggest that I should write a script that removes these file from the TZ-TRNG porject?

Reply:
I was just curious about the integration. It better would be to use a script for updates - otherwise we end up with lot of unused code in mbed-os. This is my only concern here

Reply:
Thanks for your clarification. I will make a script that, from the release driver, deletes unused files by mbed-os. Only the used files will be in mbed-os repository, and that script to show people how it was done 👍